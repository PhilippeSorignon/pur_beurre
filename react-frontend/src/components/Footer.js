import React from "react";

class Footer extends React.Component {
  render() {
    return (
      <footer className="bg-light py-5">
        <div className="footer-container">
          <div>
            <a href="/">Mentions LÃ©gales</a>
          </div>
          <div>
            <a href="/">Contact</a>
          </div>
        </div>
      </footer>
    );
  }
}

export default Footer;
