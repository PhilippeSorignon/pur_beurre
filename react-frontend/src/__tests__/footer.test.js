import React from 'react';
import ReactDOM from 'react-dom';
import Footer from '../components/Footer';

import { cleanup } from '@testing-library/react';
import renderer from "react-test-renderer";

afterEach(cleanup);

it("Footer renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Footer />, div);
});

it("Footer matches snapshot", () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
})
