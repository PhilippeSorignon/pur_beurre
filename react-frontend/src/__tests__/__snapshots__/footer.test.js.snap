// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<footer
  className="bg-light py-5"
>
  <div
    className="footer-container"
  >
    <div>
      <a
        href="/"
      >
        Mentions LÃ©gales
      </a>
    </div>
    <div>
      <a
        href="/"
      >
        Contact
      </a>
    </div>
  </div>
</footer>
`;
